### PR TITLE
style: ESLint & Prettier

### PR DESCRIPTION
**Pull request by Rapsssito**

### CODE REVIEW

**Comment by Rapsssito** [comment]
` yarn.lock` is added for development purposes. I can remove it from the library with the `.npmignore` if you prefer so.

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "tabWidth": 4,
+    "semi": true,
```
Comment (mvayngrib) [inline .prettierrc]: please stay close the currently implied settings

Comment (Rapsssito) [reply .prettierrc]: Updated to implied settings.

**Comment by Rapsssito** [comment]
@mvayngrib all changes are ready!

**Pull request merged**